packages/datadog-plugin-aws-sdk/src/services/stepfunctions.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+const log = require('../../../dd-trace/src/log')
+const BaseAwsSdkPlugin = require('../base')
+
+class Stepfunctions extends BaseAwsSdkPlugin {
+  static get id () { return 'sfn' }
```

Comment:
```suggestion
  static get id () { return 'stepfunctions' }
```

This is the [canonical service name for step functions](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/stepfunctions/index.html) at least in `awscli` (for AWS' definition of service) AFAICT, so it might be more meaningful to users.